docs/reference/transform/examples.asciidoc

```diff
@@ -336,3 +337,116 @@ This {transform} makes it easier to answer questions such as:
 * Which client IPs have high error rates?
 
 * Which client IPs are interacting with a high number of destination countries?
+
+
+[[example-last-log]]
+== Finding the last log event from an IP address
+
+This example uses the web log sample dataset to find the last log from an IP 
+address. Let's use the `latest` type of {transform} which copies the most recent 
+documents from the source index to the destination index. Pick the `clientip` 
+field as the unique key; the data is grouped by this field. Select `@timestamp` 
+as the date field that sorts the data chronologically. The {transform} creates 
+the destination index that contains the latest login date for each client IP.
```

Comment:
I think its hard to sell this feature without running in continuous mode.  We also need some reason why: 

With latest you will be able to quickly get the current(latest) state of all your entities, independent of when the last update occurred. On top of latest you can run further summarizations quick and efficient.